bundles/org.openhab.binding.knx/src/main/resources/OH-INF/thing/serial.xml

```diff
@@ -13,6 +13,13 @@
 				<label>Serial Port</label>
 				<description>The serial port to use for connecting to the KNX bus</description>
 			</parameter>
+			<parameter name="useCemi" type="boolean">
+				<label>Use CEMI</label>
```

Comment:
Please also update knx.properties
You can use the specific maven command.

Reply:
@lolodomo Can you pls. give more details? I am not aware yet how to do this....

Reply:
got it, `mvn org.openhab.core.tools:i18n-maven-plugin:3.2.0:generate-default-translations -pl :org.openhab.binding.knx`

Reply:
Perfect